dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryContainerFilter.java

```diff
@@ -129,27 +130,42 @@ public String get(final ContainerRequestContext carrier, final String key) {
     private static class ServerAttributesExtractor
             implements HttpServerAttributesGetter<ContainerRequestContext, ContainerResponseContext> {
 
+
+        private static final String ROUTE = "route";
```

Comment:
This should be with the other property names above in the class.  Likely it should be resourceString + "route".  In general configString and resourceString should both be static and final.  Additionally the actual properties all should be static final there and referenced in filter and route instead of doing string concatenation on ever filter and route call.

Reply:
I think I have addressed this now.